kubetest/anywhere.go

```diff
@@ -131,6 +147,25 @@ func newKubernetesAnywhere(project, zone string) (deployer, error) {
 	return k, nil
 }
 
+func resolveCIVersion(version string) (string, error) {
+	if strings.HasPrefix(version, "v") {
+		return version, nil
+	}
+	file := fmt.Sprintf("gs://kubernetes-release-dev/ci/%v.txt", version)
+	return readGSFile(file)
+}
+
+// Implemented as a function var for testing.
```

Comment:
Consider inlining this with the var line?

Reply:
If it is ok with you, I would like to tackle that when I tackle the TODO on figuring out how to restore the original functionality of this var after tests.